### PR TITLE
Fix: ignite

### PR DESCRIPTION
**Pull request by pr0n00gler**
Related PRs:
* https://github.com/neutron-org/neutron-dev-contracts/pull/36

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -26,6 +26,7 @@ require (
 	github.com/golang/protobuf v1.5.3
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.18.1
```
Comment (sotnikov-s) [inline go.mod:29]: don't see where it is needed. a misprint?

**Review by swelf19 — COMMENTED** [review]
lets add `openapi generate` command into the Makefile
and add a check into gh workflow that checks openapi is uptodated
what do you think.

File: go.mod (hunk 1 above)
Comment (mmikev) [reply go.mod:29]: Forgot to push a file

**Comment by pr0n00gler** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6957514732

**Comment by pr0n00gler** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6972677732

**Review by sotnikov-s — APPROVED** [review]

**Pull request merged**